libmscore/volta.cpp

```diff
@@ -327,7 +327,7 @@ void Volta::setChannel() const {
             Staff* st = staff();
             for (int voice = 0; voice < VOICES; ++voice) {
                   int channel = st->channel(startTick, voice);
-                  st->channelList(voice)->insert(endTick, channel);
+                  st->insertIntoChannelList(voice, endTick, channel);
```

Comment:
this isn't about adding mtest, is it? More to fix the 1st commit's [compile error](https://travis-ci.org/musescore/MuseScore/jobs/451909849#L2330-L2333), right?